rsmas_move_outfiles.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3 
+
+import shutil
+import glob
+import os
+import argparse
+import sys
+
+def create_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("dataset", metavar="DATASET NAME", help="the dataset whos out_* files to move")
+
+    inps = parser.parse_args(sys.argv[1:])
+
+    return inps
```

Comment:
nit: you can just refactor to `return parser.parse_args(sys.argv[1:])`

Reply:
Hmm, I don't know if I was clear. I mean you could combine those 2 lines into 1 line since you're just assigning a variable and then returning it. So instead of instantiating, you could combine it into one line.

Reply:
I understood. I left the variables name as inps, so no one get confused at to what that parser object actually is relative to the rest of the codebase. At some point I would like to standardize the way we write argument parsing in terms of variables names and syntactical conventions, but that is for another day.